packages/DataGrid/src/DataGrid.js

```diff
@@ -528,7 +528,7 @@ const DataGrid = React.forwardRef((props, ref) => {
       </sc.Grid>
       <sc.Footer $width={gridWidth}>
         <sc.RowCount>
-          Rows:{rowCount} Columns:{columnCount}
+          {i18n.t("dataGrid.rows")}:{rowCount} {i18n.t("dataGrid.columns")}:{columnCount}
```

Comment:
I think it would be better to interpolate the counts into the i18n key.  The `:` is probably not used for every locale, or the spacing may vary – for example in English there should be space after it, like "Rows: 12", but in French, there should be a space before too, like "Les lignes : 12".